content/engine/release-notes/25.0.md

```diff
@@ -19,6 +19,54 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](../api/version-history.md).
 
+## 25.0.2
+
+{{< release-date date="2024-01-31" >}}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 25.0.2 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A25.0.2)
+- [moby/moby, 25.0.2 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A25.0.2)
+
+### Security
+
+This release contains security fixes for the following CVEs
+affecting Docker Engine and its components.
+
+| CVE                                                         | Component     | Fix version | Severity         |
+| ----------------------------------------------------------- | ------------- | ----------- | ---------------- |
+| [CVE-2024-21626](https://scout.docker.com/v/CVE-2024-21626) | runc          | 1.1.12      | High, CVSS 8.6   |
+| [CVE-2024-23651](https://scout.docker.com/v/CVE-2024-23651) | BuildKit      | 1.12.5      | High, CVSS 8.7   |
+| [CVE-2024-23652](https://scout.docker.com/v/CVE-2024-23652) | BuildKit      | 1.12.5      | High, CVSS 8.7   |
+| [CVE-2024-23653](https://scout.docker.com/v/CVE-2024-23653) | BuildKit      | 1.12.5      | High, CVSS 7.7   |
+| [CVE-2024-23650](https://scout.docker.com/v/CVE-2024-23650) | BuildKit      | 1.12.5      | Medium, CVSS 5.5 |
+| [CVE-2024-24557](https://scout.docker.com/v/CVE-2024-24557) | Docker Engine | 25.0.2      | Medium, CVSS 6.9 |
+
+The potential impacts of the above vulnerabilities include:
+
+- Unauthorized access to the host filesystem
+- Compromising the integrity of the build cache
+- In the case of CVE-2024-21626, a scenario that could lead to full container escape
+
+For more information about the security issues addressed in this release,
+refer to the [blog post](https://www.docker.com/blog/docker-security-advisory-multiple-vulnerabilities-in-runc-buildkit-and-moby/).
+For details about each vulnerability, see the relevant security advisory:
+
+- [CVE-2024-21626](https://github.com/opencontainers/runc/security/advisories/GHSA-xr7r-f8xq-vfvv)
+- [CVE-2024-23651](https://github.com/moby/buildkit/security/advisories/GHSA-m3r6-h7wv-7xxv)
+- [CVE-2024-23652](https://github.com/moby/buildkit/security/advisories/GHSA-4v98-7qmw-rqr8)
+- [CVE-2024-23653](https://github.com/moby/buildkit/security/advisories/GHSA-wr6v-9f75-vh2g)
+- [CVE-2024-23650](https://github.com/moby/buildkit/security/advisories/GHSA-9p26-698r-w4hx)
+- [CVE-2024-24557](https://github.com/moby/moby/security/advisories/GHSA-xw73-rw38-6vjc)
+
+### Packaging updates
+
+- Upgrade containerd to [v1.6.28](https://github.com/containerd/containerd/releases/tag/v1.6.28).
```

Comment:
Wasn't this only for Moby 23?

Reply:
1.7.13 is used only in CI/static packages. For the runtime usage we still package 1.6.

cc @thaJeztah correct me if I'm wrong please.

Reply:
I came across this note on the wiki checklist:

> moby/moby vendor: github.com/containerd/containerd v1.7.13 / v1.6.28 (TBD: containerd 1.6 has updated OTEL versions, and other changes not directly related to this CVE, so we can keep this for v25.0.3)

Reply:
We didn't update containerd vendoring, but;

- release packages (deb / rpm) for containerd 1.6.28 (includes runc 1.1.12)
- static packages contain containerd 1.7.13 (and runc 1.1.12)